app/models/comfy/cms/page.rb

```diff
@@ -82,4 +82,8 @@ def suppress_mirrors_from_links_recirculation
   def ever_been_published?
     revisions.map {|revision| revision.data[:event]}.include?('published')
   end
+
+  def publishable?
+    keywords.map{ |tag| tag.value }.exclude?('do-not-publish')
```

Comment:
I agree, but there are two concerns here. The `#publishable?` thing has meaning for `Page`, which will still be around, it just essentially delegates its logic to `Tag#(publishable?|embargoed?)` whichever which way you decide to implement it.

Perhaps:

``` ruby
# Page#publishable?
def publishable?
  tags.all?(&:publishable?)
end

# Tag#publishable?
def publishable?
  value == 'blah-what-ever'
end
```